app/src/main/java/com/wednesday/banklocator/network/RetrofitInstance.kt

```diff
@@ -8,6 +8,8 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 class RetrofitInstance {
     companion object {
+        
+        const val baseUrl:String = "https://ifsc.razorpay.com/"

```

Comment:
Use type inference. You don't need to declare the type when you're defining it.

Reply:
sure thing!